migration-db.md

```diff
@@ -0,0 +1,148 @@
+# Migrate a PostgreSQL Database From One Environment to Another Using Elastio
+
+A common problem, slowing down the SDLC (Software Development Life Cycles, occurs when the databases in the different environments (dev, qa, staging, and prod) are not in sync. And, without an automated process of sorts, DBAs will battle to ensure that every environment's databases are always synchronized. 
```

Comment:
I do not understand this sentence: "And, without an automated process of sorts"
Do we really want to use "sorts" here?